package.json

```diff
@@ -22,6 +22,7 @@
     "generate": "run-s generate:*",
     "generate:imports": "node tools/generate-imports.mjs",
     "git-check": "node tools/check-git-version.mjs",
+    "generate-hash": "node tools/generate-hash.mjs",
```

Comment:
@viceice is `generate` now essentially just an alias for `generate:imports` or is there something else I'm missing?

@RahulGautamSingh does `generate-hash` depend on either of the steps before it or in theory could it run as part of `generate:*`?

Reply:
no, the import need to be generated before compile. maybe we should have some pre and post generate scripts?

Reply:
@rarkins What about fully generating `fingerprint.ts`, like we to for the json imports file?

Reply:
So it would be regenerated every time we run "yarn start .. "?